opencv_giflib.cpp

```diff
@@ -212,8 +217,29 @@ bool giflib_decoder_decode(giflib_decoder d, int frame_index, opencv_mat mat) {
     // when encoding back to gif, so that the resized frame is drawn to the
     // correct location
 
+    // skip entire rows at the top if frame_top < 0
```

Comment:
this is where we start looking at drawing *this* frame and moving the raster bits pointer around